src/model/server.ts

```diff
@@ -41,9 +44,39 @@ function parseOrdDirectory(ordDirectory: string | undefined, sourceType: OptSour
   return ordDirectory;
 }
 
+interface MtlsTrustedCertsConfig {
+  certs: { issuer: string; subject: string }[];
+  rootCaDn: string[];
+  configEndpoints?: string[];
+}
+
+function parseMtlsTrustedCerts(
+  value: string | undefined,
+): { certs: { issuer: string; subject: string }[]; rootCaDns: string[]; configEndpoints: string[] } | undefined {
+  if (!value || value.trim() === "") {
+    return undefined;
+  }
+
+  try {
+    const parsed = JSON.parse(value) as MtlsTrustedCertsConfig;
+
+    return {
+      certs: parsed.certs,
+      rootCaDns: parsed.rootCaDn,
+      configEndpoints: parsed.configEndpoints || [],
+    };
```

Comment:
I think some primitive validation of input values would be valuable here. Users can put any non-sense, e.g. we need to ensure at least on the types level: `certs` must be an array of objects of a certain kind, `configEndpoints` is an array of URLs, `rootCaDns` is an array of strings..

Reply:
I have tried to start a server with:
```
CF_MTLS_TRUSTED_CERTS = {"certs":[],"rootCaDn":["CN=Root CA,O=ACME,C=DE"]}
```

But still got the following error:
```
   2025-12-05T20:59:54.47+0100 [APP/PROC/WEB/0] OUT {"level":50,"time":1764964794478,"pid":8,"hostname":"65703ff5-547c-4af5-4d8a-d130","msg":"ValidationError: \n- CF_MTLS_TRUSTED_CERTS.certs must be an array"}
```

It's an array, just an empty array.